Explorer/Assets/DCL/UI/Profile/ProfileElements/ProfileSectionController.cs

```diff
@@ -51,6 +53,14 @@ private async UniTask LoadAsync(CancellationToken ct)
             //temporarily disabled the profile image request untill we have the correct
             //picture deployment
             //await profileImageController!.RequestImageAsync(profile.Avatar.FaceSnapshotUrl, ct);
+            LayoutRebuilder.ForceRebuildLayoutImmediate(element.RectTransform);
```

Comment:
Is this really needed? It's pretty heavy :(

Reply:
or is it called only one time? in that case it's okay I guess

Reply:
Yes, its horrible, but I tried like 20 different options and it was either this or waiting until the end of frame with an Unitask and do some change on the Layout anyway... The issue is that basically we got a vertical layout, that contains amongst other things an horizontal layout whose width can change (depending on the name of the user), and it needs to be centered. Without this line, it doesnt get centered the first time the name is set (but its correctly set the following times). So the width of the horizontal layout is correct, but its not centered with the rest of the elements.
I can although try other options xD this one suggestion by Santi worked after some tuning of the prefabs but maybe there is a less costly one as sadly this is run every time we show the menu, as we need to display latest information from profile.

Reply:
Shouldn't be needed. Let's take a quick look together to see if there is any other solution 👍